developer/general/gsod.md

```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
-- Propose updated and clarified content
+    The Disposable VM documentation is consilidated into a few pages and easier to find.
 
-**Knowledge prerequisite**:
+    Disposable VM terminology is consistent across all pages
 
-- basic Qubes OS knowledge - [intro], [getting started]
-- [Markdown][markdown]
+    There is a new installation guide detailing how to install Qubes on virtual machines (VirtualBox), complete with screenshots and troubleshooting steps where necessary. 
 
-**Mentor**: [Marek Marczykowski-Górecki][team]
+    There is a decrease (by ~20%) in the number of issues and questions raised for topics covered in the documentation, whether in the official Github issues tracker, the Qubes mailing list, or the Qubes community forum.
 
-### Installation Guide for Qubes OS on Virtual Machines
+#### Budget
 
-**Project**: Installation Guide for Qubes OS on Virtual Machines.
+| Budget item | Amount | Running total | Note / Justification |
+|-|-|-|-|
+| Technical writer audit, update, and write new documentation | $12000 | $12000 | based on a 20hr/week for 6 months at 25$/hr |
+| TOTAL |  | $12000 |  |
 
-**Brief explanation**: The Qubes OS is missing an installation guide for virtual machines. Users are installing an outdated and unsupported version of Qubes OS (3.2) instead of the supported version. There is unofficial [existing installation guide] for Qubes OS on a virtual box but it is misleading and lacks documentation. Usually, users face some errors and bugs while installing Qubes OS on a virtual machine.[virtual box issue]
+#### Additional information 
+Qubes OS regularly participates in the Google Summer of Code and Google Season of Docs. This is our third time in a row participating in Google Season of Docs. Our mentorships for GSoD 2019 and 2020 were successes and both projects were completed 
+within the times allotted. The past Google Season of Docs projects have given us experience in working with technical writers, and has helped us to understand the benefits of technical writers can to our project
 
-**Expected results**: 
- -Provide a new option of installation guide for users working on virtual machines.
- -Review existing problems and provide solutions to them.
- -Giving a warning for using outdated versions.
-
-**Knowledge prerequisite**: 
- - Experience in virtual boxes and machines.
- - Basic Knowledge about Fedora linux architecture.
- - [Markdown][markdown]
- 
-**Mentor**: [Marek Marczykowski-Górecki][team]
+You can view the project we had in 2019 in the [2019 GSoD archive](https://developers.google.com/season-of-docs/docs/2019/participants/project-qubes) and the [2019 writer’s report](https://refre.ch/report-qubesos/).
+You can also view the project we had in 2020 in the [2020 GSoD archive](https://developers.google.com/season-of-docs/docs/2020/participants/project-qubesos-c1e0) and the [2020 writer’s report](https://gist.github.com/PROTechThor/bfe9b8b28295d88c438b6f6c754ae733).
```

Comment:
We can remove this section, since it's already available below under "Past Projects"